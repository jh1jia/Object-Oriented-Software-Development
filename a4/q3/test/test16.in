test16.txt count 9
