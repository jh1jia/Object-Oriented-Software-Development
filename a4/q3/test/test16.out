1111111111
2222222222
3333333333
0123456781
2876543210
7861032364
1
2
3
4
5
6
7
0
4
